### PR TITLE
Feature/telemetry configuration grafana integration

### PR DESCRIPTION
**Pull request by manoj22-k**
Created Configuration page for Telemetry API
Embedded Grafana inside UI 
Added Backend and UI UTs

### CODE REVIEW

File: src/mtool/ui/src/locales/translation_kor.json [hunk 1]
```diff
@@ -2,5 +2,9 @@
     "Username" :"아이디 또는 이메일 아이디",
     "Password" : "암호",
     "Login" : "로그인",
-    "Login failed! Invalid id or password" : "로그인 실패! ID나 비밀번호가 잘못되었습니다."
+    "Login failed! Invalid id or password" : "로그인 실패! ID나 비밀번호가 잘못되었습니다.",
+    "IP Address": "IP 주소",
+    "Port": "포트",
+    "Save": "득점 방해",
```
Comment (mjlee34) [inline src/mtool/ui/src/locales/translation_kor.json:8]: "득점 방해" means "Preventing scores" in Korean. 
Here, what does "Save" mean?

**Comment by mjlee34** [comment]
> Created Configuration page for Telemetry API Embedded Grafana inside UI Added Backend and UI UTs

Hi, thank you for the contribution. 

For better review process, I suggest you to explain the pull request in more detail. 
For example, I think you can explain the purpose of PR, what configuration here means, and so on. 

Because this PR is about pos-gui, figures can help a lot.

File: src/mtool/ui/src/locales/translation_kor.json (hunk 1 above)
Comment (manoj22-k) [reply src/mtool/ui/src/locales/translation_kor.json:8]: Hi, here "Save" means to store the data. Can "저장" be used?

Comment (mjlee34) [reply src/mtool/ui/src/locales/translation_kor.json]: I think "저장" is appropriate here.

File: src/mtool/ui/src/containers/Authentication/index.test.js [hunk 2]
```diff
@@ -92,106 +92,226 @@ describe("Authentication", () => {
...
+    fireEvent.click(submitLogin)
   });
 
+  it("Edit Configuration", async () => {
+    const mock = new MockAdapter(axios);
+    const { getByTestId } = wrapper;
+
+    const telemetryIPInput = getByTestId("telemetryIPInput").querySelector("input");
+    fireEvent.change(telemetryIPInput, {
+      target: { value: '107.108.83.97', name: 'telemetryIP' }
```
[the remark this answers is not in the document]
Comment (manoj22-k) [reply src/mtool/ui/src/containers/Authentication/index.test.js]: I have changed the IP to 127.0.01, thanks

**Review by mjlee34 — APPROVED** [review]
LGTM

**Review by aswinssir — APPROVED** [review]
LGTM :shipit:

**Pull request merged**